src/pyfme/aircrafts/cessna_172.py

```diff
@@ -307,9 +314,9 @@ def _calculate_aero_forces_moments(self):
 
         return L, D, Y, l, m, n
 
-    def _calculate_thrust_forces_moments(self):
+    def _calculate_thrust_forces_moments(self, environment):
         delta_t = self.controls['delta_t']
-        rho = self.rho
+        rho = environment.rho

```

Comment:
Same goes for atmospheric properties, that now belong to `Environment`. Correct?